apps/developer-hub/src/components/Case/index.tsx

```diff
@@ -0,0 +1,171 @@
+export const CASE_VARIANTS = [
```

Comment:
So the problem with modules like this is that they are extremely locale specific and they make it a nightmare to localize codebases if we ever decide to do so.  I generally recommend explicitly mapping strings rather than relying on clever tricks like this because mapping strings explicitly is simpler, drastically easier to localize, and also generally less brittle.